spec/config/test_setup_migration.rb

```diff
@@ -0,0 +1,18 @@
+class TestSetupMigration < ActiveRecord::Migration
+  def up
+    return if ActiveRecord::Base.connection.table_exists? :test_classes
+
+    create_table :test_classes do |t|
+      t.string :field
+    end
+
+    create_table :association_test_classes do |t|
+      t.integer :test_class_id
+    end
+  end
+
+  def down
```

Comment:
We don't; it's just nice to be able to undo the migration. I modeled this code heavily off of what we do in [unique_attributes](https://github.com/panorama-ed/unique_attributes/tree/master/spec).